### ISSUE TITLE
Replace using regex

### ISSUE THREAD

**Issue by 7othifaPS** [SkriptLang/Skript#6547]
### Suggestion

Add removing regex matches from string with syntax

`replace all "[a-z]" matches in {_string} with "1"`

_can have better syntax, or the same syntax for replacing but make it work with regex_

### Why?

To make stuff much easier rather than needing skript-reflect for it

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by sovdeeth** [comment]
Yes, there's already a PR for this that's nearly finished. #4323

**Comment by 7othifaPS** [comment]
alright nice

waiting for it to be added......
